### ISSUE TITLE
Email editor is not accessible to screen readers

### ISSUE THREAD

**Issue by legle** [mautic/mautic#12263]
### Mautic Version

4.4.x series

### PHP version

7.4

### What happened?

I'm a blind person and when I am editing emails on mautic email editor is not possible to identify richtext area and is not possible to edit an email.
the unic way to edit an email for a blind person is a html editor and it is not a very confortable way to edit emails.
it is possible to fix this issue for us?

### How can we reproduce this issue?

1. start a screen reader like NVDA or JAWS;
2. go to the emails section;
3. clic on new email button;
4. enter normal information t your new email;
5. After this on new email screen or edit email screen click on email editor button;
6. after this start exploring the email editor screen with arrow keys and observ thath the rich area is not reported and is not accessible to screen reader.

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

**Comment by legle** [comment]
Aditional information: this problem ocurs on versions 2 and 3 too.
